clean-up-migrations.md

```diff
@@ -34,9 +34,13 @@ rm -rf migrations/*
 ```
 
 Now, dump the current schema into a new single migration:
-
+> [WARNING] This method don`t dump foreign keys constraints.
 ```terminal
 symfony console doctrine:migrations:dump-schema
+```
+ Instead use:
+ ```terminal
+symfony console doctrine:migrations:diff --from-empty-schema
 ```
```

Comment:
And here write this instead:

```suggestion
***TIP
The `doctrine:migrations:dump-schema` command may lead to losing foreign key constraints. Instead, use:
\ ```terminal
symfony console doctrine:migrations:diff --from-empty-schema
\```
***
```

That's our internal syntax for notes